src/Components/AddBook/AddBook.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from 'react'
 import './AddBook.css'
-import { gql, useLazyQuery } from '@apollo/client'
```

Comment:
Seeing this code progress, it feels like the useLazyQuery fixed one problem and added another.